queue/index.php

```diff
@@ -1,46 +1,65 @@
 <!DOCTYPE html>
-<html class="no-js" lang="">
+<html class='no-js theme-light' lang="">
     <head>
-        <title>Queue Test</title>
+		<title>Queue Test</title>
+		<?php require_once '../server/header.php' ?>
 		<link rel='stylesheet' href='../assets/css/queue.css'>
+		<meta http-equiv='refresh' content='600'/>
     </head>
     <body>
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
-
-		<div class='vita-header'>
-			<div class='vita-volunteer-count'>Volunteers on duty: 0</div>
-			<img class='vita-header-curtain' src='../assets/res/queue_curtain.svg'>
-			<img class='vita-exit-fullscreen' src='../assets/res/generic_exit.svg'>
+		<![endif]-->
+		<?php require_once '../components/nav.php' ?>
```

Comment:
is there any reason this happens down below the ie stuff, or can this happen up by the other php require_once?